ui/cypress/e2e/checks.test.ts

```diff
@@ -160,4 +160,51 @@ describe('Checks', () => {
       cy.getByTestID('inline-labels--empty').should('exist')
     })
   })
+
+  describe('Alert history page loads after check created after set duration', () => {
+    beforeEach(() => {
+      // creates a check before each iteration
+      // TODO: refactor into a request with other before each
```

Comment:
https://github.com/influxdata/influxdb/issues/18454

Im going to go make an issue for this to be tackled, but im also going to write up issues for a few more of our todos. This TODO is specifically is in like three places, so we need to catalog where it will go and such.

Reply:
Way to go!